netket/graph/lattice.py

```diff
@@ -103,14 +103,14 @@ def create_sites(
     return sites, is_inside, coord_to_site
 
 
-def get_edges(positions, cutoff):
+def get_edges(positions, cutoff, order):
     kdtree = cKDTree(positions)
     dist_matrix = kdtree.sparse_distance_matrix(kdtree, cutoff)
     row, col, dst = find(triu(dist_matrix))
     dst = comparable(dst)
     _, ii = _np.unique(dst, return_inverse=True)
 
-    return sorted(list(zip(row[ii == 0], col[ii == 0])))
+    return [sorted(list(zip(row[ii == k], col[ii == k]))) for k in range(order)]
 
 
 def get_true_edges(
```

Comment:
I know it was not written now, but while we are at it could we get a docstring/documentation about what this function does and it's arguments?

Reply:
Okay, I can write something.